### ISSUE TITLE
Primes as a Dependency

### ISSUE THREAD

**Issue by djourd1** [tobydriscoll/HaltonSequences.jl#3]
Using the HaltonSequences under Julia 1.1 I am having the following warning:

Warning: Package HaltonSequences does not have Primes in its dependencies:
│ - If you have HaltonSequences checked out for development and have
│   added Primes as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with HaltonSequences
└ Loading Primes into HaltonSequences from project dependency, future warnings for HaltonSequences are suppressed.

The suggested Pkg.resolve() did not really solve anything.
Is it me, or an issue?

**Comment by tobydriscoll** [comment]
Not sure. It may be my fault on the packaging. 

Try updating, which should delete the Manifest.toml file, and see if it's happier.

**Comment by djourd1** [comment]
The resolve() does recreate a Manifest.toml. However, the dependencies on Primes is not there. Here is the section dedicated to HaltonSequences:

[[HaltonSequences]]
deps = ["Test"]
git-tree-sha1 = "e08ec73f4f7cf01c626681926a720cc7aa3416f8"
uuid = "13907d55-377f-55d6-a9d6-25ac19e11b95"
version = "0.1.0"

**Comment by djourd1** [comment]
Sorry, I also did an update() with the same result.

**Comment by tobydriscoll** [comment]
I don't really understand what happened, but I have been able to reproduce and clear it up on a local copy. Within the package interface (`]` at the command line), try this:

```julia
rm HaltonSequences
add https://github.com/tobydriscoll/HaltonSequences.jl
test HaltonSequences
```
Let me know if that works for you, before I trigger an official version update at the registry (at which point you can go back to adding it normally).

**Comment by djourd1** [comment]
This version works and does not trigger the warning message anymore.

**Comment by binyamink** [comment]
Had the same problem and the above solution helped. Thanks for the package!

**Comment by jwscook** [comment]
Was this fix intended to go out in the release 0.1.0? I still get this error from `[13907d55] HaltonSequences v0.1.0` via `] add HaltonSequences`, but will `add https://github.com/tobydriscoll/HaltonSequences.jl`.
